src/BUILD

```diff
@@ -129,6 +133,18 @@ cc_library(
     ],
 )
 
+cc_binary(
```

Comment:
You didn't add dependency for ovms_test target on that library.

This means that you cannot run just:
bazel --test_summary=detailed --test_output=all --test_filter=** //src:ovms_test
and expect all tests to pass.

Reply:
Added dependency of "//src:libsampleloader.so", for test